radicle-node/src/service.rs

```diff
@@ -124,15 +122,15 @@ pub enum Command {
 pub enum CommandError {}
 
 #[derive(Debug)]
-pub struct Service<A, S, G> {
+pub struct Service<R, A, S, G> {
     /// Service configuration.
     config: Config,
     /// Our cryptographic signer and key.
     signer: G,
     /// Project storage.
     storage: S,
-    /// Tracks the location of projects.
-    routing: Routing,
+    /// Network routing table. Keeps track of where projects are located.
+    routing: R,
```

Comment:
I’d use a concrete type here for simplicity but you may have more context on what’s going to happen with this portion of code in the near term